src/List/ContextActions.js

```diff
@@ -9,14 +9,19 @@ config.i18n.strings.add('assignToOrgUnits');
 
 const contextActions = Action.createActionsFromNames([
   'details',
-  'assignToOrgUnits'
+  'assignToOrgUnits',
```

Comment:
Do you see the icons in the app? I needed to change all "*" to "**" in webpack.config.js proxy routes for the proxy to work (https://github.com/webpack/webpack-dev-server/issues/458#issuecomment-219297105)